src/app/code/community/Zendesk/Zendesk/Block/Adminhtml/Dashboard/Tab/Tickets/Grid/Renderer/Email.php

```diff
@@ -19,20 +19,24 @@
 class Zendesk_Zendesk_Block_Adminhtml_Dashboard_Tab_Tickets_Grid_Renderer_Email extends Mage_Adminhtml_Block_Widget_Grid_Column_Renderer_Abstract {
 
     public function render(Varien_Object $row) {
-        $users = Mage::registry('zendesk_users') ? Mage::registry('zendesk_users') : array();
+        $users = Mage::registry('zendesk_users');
         $value = (int) $row->getData($this->getColumn()->getIndex());
-        
-        $found = array_filter($users, function($user) use($value) {
-            return (int) $user['id'] === $value;
-        });
-        
-        if( count($found) ) {
+
+        if ($users) {
```

Comment:
As closure functions are supported from PHP >= 5.3.0, I would discourage to use them within Magento code as they are not used anywhere else. It does make use of `create_function()` or you could just simply iterate over the array with foreach and `return $user['email']` when the right user ID was found.